SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -257,20 +256,28 @@ public class SwiftyTesseract {
     setenv(variableName.rawValue, value, 1)
   }
 
-  private func processPDF(images: [UIImage], at filepath: URL) throws {
+  private func processPDF(images: [UIImage], at filepath: URL? = nil) throws -> URL {
+    let path = validate(filepath)
+    let renderer = try makeRenderer(at: path)
 
-    let renderer = try makeRenderer(at: filepath)
-    
     defer {
       TessDeleteResultRenderer(renderer)
     }
-    
+
     try render(images, with: renderer)
+    return path.appendingPathComponent("pdf")
```

Comment:
One of the reasons the tests are still failing is this is creating a file path of `file://some/path/to/uuid/pdf` and we need to it to be `file://some/path/to/uuid.pdf`. So we'll need to use `addingPathExtension(_:)` and not `addingPathComponent(_:)`.